content/reference/v1beta1/config.md

```diff
@@ -225,6 +225,10 @@ Items can be hidden. They will not be visible if hidden.
           value: "{{repl RandomString 40}}"
 ```
 
+When used in conjunction with [RandomString](/reference/template-functions/static-context/#randomstring)
```

Comment:
its my experience that readonly affect this. does hidden?

Reply:
It doesn't but it generally is used with `hidden` If its not a hidden field they can edit it.